### DISCUSSION TITLE
Have you considered 3D and using dot to represent function and a big dot to represent popular function similar to ant colony?

### DISCUSSION THREAD

**Discussion by hdc1112** [chanhx/crabviz#46] [Ideas]
I had a similar idea in my mind, and searched your reddit article https://www.reddit.com/r/rust/comments/142is0h/crabviz_a_call_graph_generator_for_various/, and reached here.

Basically the idea is to revolutionize the UML and visualize the call graph in 3D similar to ant colony structure. The product's goal is to kill tribal knowledge and speed up on-boarding process for developers.

**Comment by chanhx** [comment]
Sorry for the late reply.
I've seen something like that before, and the 3D call graph made me feel dizzy. I think it’s not as easy to understand as ordinary call graphs, and it can’t be shared easily in any image format. So I probably wouldn’t consider 3D call graphs, but any way thank you for sharing the idea.
